backend/src/mongo/models/Preference.js

```diff
@@ -0,0 +1,26 @@
+import mongoose from 'mongoose';
+
+const preferenceSchema = mongoose.Schema({
+  sessionId: {
+    type: String,
+    required: true,
+  },
+  location: {
+    type: String,
+    required: true,
+  },
+  distance: {
+    type: Number,
+    required: true,
+  },
+  cuisines: {
+    type: [String],
+    default: [],
+  },
+  price: {
+    type: Number,
+    required: true,
+  },
```

Comment:
I like your eagerness to get into the code, but can we just make sure your api contract is finished and approved before pushing this?

![image](https://user-images.githubusercontent.com/49140391/111232375-e0622980-864f-11eb-98ad-227a27234ddf.png)

Reply:
As far as I can tell it looks correct, but just create a sample payload using mock data so everyone has access to the same test case

Reply:
Done.